apps/src/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon.tsx

```diff
@@ -38,7 +38,7 @@ export interface FontAwesomeV6IconProps {
  * Can can be used in any component in/out of the scope of Design System.
  */
 const FontAwesomeV6Icon: React.FunctionComponent<FontAwesomeV6IconProps> = ({
-  iconStyle,
+  iconStyle = 'solid',
```

Comment:
@levadadenys are you cool with me adding a default here? Is `solid` the right choice, or something else?